client/src/pages/guide/english/c/arrays-and-strings/index.md

```diff
@@ -162,6 +162,17 @@ Notice the `!`, which is needed because this function returns 0 if they are the
 char *strtok(char *str, const char *delim);
 ```
 
+#### Copy block of memory: `Memcpy` 
+
+`void * memcpy ( void * destination, const void * source, size_t num );`
+
+Copies the values of num bytes from the location pointed to by source directly to the memory block pointed to by destination.The underlying type of the objects pointed to by both the source and destination pointers are irrelevant for this function.
```

Comment:
Space after the dot. But more importantly, the two memory areas must not overlap. It's required both by the C standard and POSIX.